book/src/user/mining.md

```diff
@@ -0,0 +1,167 @@
+# Mining Zcash with zebra
+
+Zebra's RPC methods support miners and mining pools. 
+
+Contents:
+
+- [Download and build Zebra](#download-and-build-zebra)
+- [Configure zebra for mining](#configure-zebra-for-mining)
+  - [Miner address](#miner-address)
+  - [RPC section](#rpc-section)
+- [Running zebra](#running-zebra)
+- [Testing the setup](#testing-the-setup)
+- [Run a mining pool](#run-a-mining-pool)
+
+## Download and build Zebra for mining
+[#download-and-build-zebra]: #download-and-build-zebra
+
```

Comment:
Can you link to the README section that lists the Zebra dependencies that need to be installed?
That should be the first step before building.

If we link to the list of dependencies, we will only have to update them in one place.